app/menu.js

```diff
@@ -10,7 +10,7 @@ exports.buildAppMenu = modulesPromise => {
     .reduce((menuItems, module) => {
       if (module.link) {
         return menuItems.concat({
-          label: module.name,
+          label: module.name + ` | ${module.link.split(':')[2].replace('/', '')}`,
```

Comment:
might be careful using string parsing to get out information. there are a several forms valid link url such as `http://localhost:3000/index.html`.